sycl/include/sycl/ext/intel/esimd/memory.hpp

```diff
@@ -6860,9 +6199,9 @@ void simd_obj_impl<T, N, T1, SFINAE>::copy_from(
 }
 
 template <typename T, int N, class T1, class SFINAE>
-template <int ChunkSize, typename Flags, typename AccessorT, typename TOffset>
+template <int ChunkSize, typename Flags, typename AccessorT, typename Toffset>
 ESIMD_INLINE void simd_obj_impl<T, N, T1, SFINAE>::copy_to_impl(
-    AccessorT acc, TOffset offset) const SYCL_ESIMD_FUNCTION {
+    AccessorT acc, Toffset offset) const SYCL_ESIMD_FUNCTION {
   using UT = simd_obj_impl<T, N, T1, SFINAE>::element_type;
   constexpr unsigned Size = sizeof(T) * N;
   constexpr unsigned Align = Flags::template alignment<T1>;
```

Comment:
Nit and not related to this PR but a few lines below this we have

```
 using OffsetTy = decltype(offset);
```

and we use OffsetTy. We can just using Toffset right?